src/main/java/com/manardenza/dao/UserDaoImpl.java

```diff
@@ -2,10 +2,17 @@
 
 import com.manardenza.entity.User;
 
+import java.io.File;
+import java.util.List;
+
 public class UserDaoImpl extends AbstractDao<User> {
 
     private static UserDaoImpl instance;
 
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1171)